doc/input-output-reference/src/overview/group-location-climate-weather-file-access.tex

```diff
@@ -2006,69 +2010,75 @@ \subsubsection{Outputs}\label{outputs-4-008}
 
-\paragraph{Site Wind Speed {[}m/s{]}}\label{site-wind-speed-ms}
+\subsubsection{Site Wind Speed {[}m/s{]}}\label{site-wind-speed-ms}
 
 This is the outdoor wind speed in m/s.
 
-\paragraph{Site Wind Direction {[}deg{]}}\label{site-wind-direction-deg}
+\subsubsection{Site Wind Direction {[}deg{]}}\label{site-wind-direction-deg}
 
 This is the wind direction (N = 0, E = 90, S = 180, W = 270).
 
-\paragraph{Site Sky Temperature {[}C{]}}\label{site-sky-temperature-c}
+\subsubsection{Site Sky Temperature {[}C{]}}\label{site-sky-temperature-c}
 
 The sky temperature is derived from horizontal infrared radiation intensity. It is expressed in degrees C. The default calculation is shown below but this value may be modified by the WeatherProperty:SkyTemperature object. Note that Sigma is the Stefan-Boltzmann constant in the following equation:
 
 \begin{equation}
 Sky\;Temperature = {\left( {\frac{{Horizontal\;IR}}{{Sigma}}} \right)^{\frac{1}{4}}} - {273.15_{Conversion\;from\;Kelvin\;to\;Centigrade}}
 \end{equation}
 
-\paragraph{Site Horizontal Infrared Radiation Rate per Area {[}W/m2{]}}\label{site-horizontal-infrared-radiation-rate-per-area-wm2}
+\subsubsection{Site Horizontal Infrared Radiation Rate per Area {[}W/m2{]}}\label{site-horizontal-infrared-radiation-rate-per-area-wm2}
+
+The horizontal infrared radiation intensity is expressed in \si{W/m^2} based, if missing from the weather file, on opaque sky cover, sky emissivity, temperature and other factors. The general calculation of Site Horizontal Infrared Radiation Rate per Area is discussed in both the Engineering Reference and the Auxiliary Programs document.
+
+\subsubsection{Site Diffuse Solar Radiation Rate per Area {[}W/m2{]}}\label{site-diffuse-solar-radiation-rate-per-area-wm2}
+
+Diffuse solar is the amount of solar radiation in \si{W/m^2} received from the sky (excluding the solar disk) on a horizontal surface.
 
-The horizontal infrared radiation intensity is expressed in W/m\(^{2}\) based, if missing from the weather file, on opaque sky cover, sky emissivity, temperature and other factors. The general calculation of Site Horizontal Infrared Radiation Rate per Area is discussed in both the Engineering Reference and the Auxiliary Programs document.
+\subsubsection{Site Direct Solar Radiation Rate per Area {[}W/m2{]}}\label{site-direct-solar-radiation-rate-per-area-wm2}
 
-\paragraph{Site Diffuse Solar Radiation Rate per Area {[}W/m2{]}}\label{site-diffuse-solar-radiation-rate-per-area-wm2}
+Site Direct Solar Radiation Rate per Area is amount of solar radiation in \si{W/m^2} received within a \ang{5.7} field of view centered on the sun. This is also known as Beam Solar.
 
-Diffuse solar is the amount of solar radiation in W/m\(^{2}\) received from the sky (excluding the
+\subsubsection{Site Total Sky Cover {[]}}\label{site-total-sky-cover}
 
-solar disk) on a horizontal surface.
+This is the total sky cover in tenths of sky. The value ranges from 0 (clear) to 10 (full cover). 
```

Comment:
"In tenths of sky"? With values from 0-10? Is this standard practice for this? I'd assume a "fractional" value should range from 0-1. My testing shows that this is incrementing in intervals of eighths, not tenths. Not a super important point, but just noting it.

Reply:
Yes, it is a standard practice in the weather file to use "tenths of sky" as the unit. It is value from 0 to 10 fro the weather file source.

Reply:
OK... I still think this is a confusing point for the decerning reader who really looks at it. I guess we'll just roll with it. 🙈 🙉